code/espurna/config/hardware.h

```diff
@@ -3122,14 +3122,8 @@
     #define RELAY1_PIN          15
     #define RELAY1_TYPE         RELAY_TYPE_NORMAL
 
-    // Light
-    #define LED1_PIN            0
```

Comment:
merge issue?

Reply:
right! 

Just a note: `LED1_PIN` doesn't work for 1PM and 2.5 I don't know why, if I try to change LED state I can't use `gpio 0 1` or `gpio 0 0`, because the pin doesn't have pin mode output and the led state doesn't work.

Reply:
https://github.com/xoseperez/espurna/blob/fbf2b71749307688f7258ad8235bc0f5d068e926/code/espurna/sensors/AnalogSensor.h#L72 can you remove this line?

Reply:
See here. Perhaps it is leftover from some other arduino code, since ESP only has 1 analog input pin and it is already set as INPUT by the Core setup code. This is called via sensor setup code though...
Looks like every other hw that was used to test analog sensors was not using GPIO0

Reply:
Okey! Thank you!